tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py

```diff
@@ -32,7 +32,7 @@ def launch(self):
                 *self._settings_button_locator))))
 
     def switch_to_contacts_frame(self):
-        Wait(self.marionette).until(lambda m: self.apps.displayed_app.name == self.name)
+        self.wait_to_be_displayed()
         self.apps.switch_to_displayed_app()
 
```

Comment:
I would just ignore the entry_point part for now. From what I understood from the other bug, they want to separate Phone and Contacts anyway and get rid of entry_point all together, I think.